docs/DeveloperGuide.adoc

```diff
@@ -571,24 +571,68 @@ image::implementation/BudgetPanelSequenceDiagram.png[width="800"]
 
 === Budget
 ==== Current Implementation
+The budget feature is one whereby after a user sets a budget, if the total spendings of a user exceeds his budget
+during a period determined by the user, a warning will show up to inform the user that adding an expense results in
+him exceeding his set budget.
+
 The current implementation for budget tracking is described as below.
 
-===== Keeping spendings within budget
-The budget mechanism is facilitated by `VersionedExpenseTracker` and `AddCommand`. Whenever a user adds an expense via
-`AddCommand` if they do not have a budget set, the application will prompt them to do so. After they do so,
-application proceeds normally. If adding another expense will result in them exceeding the budget, they can still do
-so, but a warning will be given to the user.
+===== Setting a budget
+Below would be how a budget is set by the user.
```

Comment:
Need to reference the sequence diagram. Maybe can change to...

Given below is a sequence diagram and step by step explanation of how the expense tracker handles setting a new budget (try to think of a better phrase than "setting a new budget". i can't think of anything)